bindings/nodejs/examples/how_tos/simple_transaction/request_funds.ts

```diff
@@ -14,18 +14,22 @@ async function run() {
         throw new Error('.env FAUCET_URL is undefined, see .env.example');
     }
     try {
+        const faucetUrl = process.env.FAUCET_URL!;
```

Comment:
`Forbidden non-null assertion`, shouldn't the check above already make sure that it's not null?

Reply:
Thats what i thought :(